### ISSUE TITLE
Add RBAC to README.md

### ISSUE THREAD

**Issue by omerozery** [wongnai/kube-slack#25]
This is just an example that works for me..

```
---
kind: ClusterRole
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: kube-slack
rules:
- apiGroups: [""]
  resources: ["pods"]
  verbs: ["get", "watch", "list"]
---
apiVersion: v1
kind: ServiceAccount
metadata:
  name: kube-slack
  namespace: kube-system
---
apiVersion: rbac.authorization.k8s.io/v1beta1
kind: ClusterRoleBinding
metadata:
  name: kube-slack
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: kube-slack
subjects:
  - kind: ServiceAccount
    name: kube-slack
    namespace: kube-system
```

**Comment by whs** [comment]
Could you please submit this as a PR?

I see @jstriebel also wrote similar RBAC in his [Helm chart](https://github.com/scalableminds/charts/tree/add-kube-slack/incubator/kube-slack/templates)

**Comment by whs** [comment]
Closed in #26
